views/forms/BarChartFields.jsx

```diff
@@ -19,7 +20,12 @@ const BarChartFields = ({
     ...prevState,
     fieldLabelValueMap: [...prevState
       .fieldLabelValueMap,
-        {  value: '', label: '', color: dark_sky_blue  }
+        {  
+          value: '', 
+          label: '', 
+          color: dark_sky_blue, 
+          targetValues: user.userAccess.map((site) => ({ site, value: '' }))
```

Comment:
🤔 this may just be a naming thing but it seems odd to be adding things to something called `userAccess` on the fly. Should this be happening "earlier" -- e.g. on the back end?

Reply:
We can have an arbitrary number of values and a user might want to enter a target value of a site for that value. 

foo value 
siteA  foo target value
siteB foo target value

baz value
siteA baz target value
siteB baz target value

Reply:
I think I'm getting it 😅 this part still confuses me
> enter a target value of a site for that value.
I thought the target value was like the value of interest for the chart, not for specific sites? so like a user would say I'm interested in `variableFoo` across all sites?

I'm not sure what the `value: ''` part does on line 27.

I'm not sure what the middle two paragraphs in your response represent, would that look something like the following? 
```js
targetValues = [
  {
    site: 'siteA',
    value: 'foo' // where 'foo' is the "value" (survey response) for a "variable" (survey question) of interest?
  },
  {
    site: 'siteB',
    value: 'foo'
  },
  {
    site: 'siteA',
    value: 'baz'
  },
  {
    site: 'siteB',
    value: 'baz'
  },
]
```

Reply:
ohhh or is more like what you posted here? https://github.com/AMP-SCZ/dpdash/pull/110#discussion_r929488244

Reply:
A user should be able to make an arbitrary number of fields on the fly and react prefers that we initialize an input field with something. https://github.com/AMP-SCZ/dpdash/pull/84#discussion_r910401176. When those fields are created they have an empty value property that initializes the value for the TextInput.

Yes the workflow is what you mentioned where variableFoo is what a user is interested in for all the sites, the variable can have different values that a user can perform a lookup for and set a label to. Then a user could add a site target for the value of that variable.